src/components/TextArea.jsx

```diff
@@ -16,16 +16,22 @@ function TextArea() {
 
```

Comment:
We could wrap all of these elements in a `<form>` element if we wanted to take advantage of default HTML to use the `required` tag on an input with a form element.

Reply:
After doing this locally for testing I found additional work will need to done to prevent this from occcuring.

We already have a fix for this in an other branch though.